.gitignore

```diff
@@ -14,5 +14,5 @@ report.xml
 *.upload
 venv/
 lib
-draft-marx-quic-logging-main-schema.xml
 .travis.yml
+.vscode/
```

Comment:
This probably should go into your global .gitignore file ;)

Reply:
lol fair